src/it/java/dev/jbang/it/RunWinIT.java

```diff
@@ -35,6 +36,7 @@ public void shouldNotLeaveVariablesCmd() {
 	}
 
 	@Test
+	@Disabled("Doesn't work in CI for some reason")
```

Comment:
No idea what's going on. I've connected to the workflow (using TMate over SSH), waited until the tests had failed and then ran the command from the test manually and of course it just worked fine :-/
So it must be something with the test itself but I haven't been able to figure out what.